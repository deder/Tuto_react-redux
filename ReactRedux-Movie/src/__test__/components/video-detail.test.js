import VideoDetail from '../../components/video-detail';
import React from 'react';

describe("<VideoDetail />",()=>{
    it("VideoDetail Avec video", () => {
        const wrapper = mount(
            <VideoDetail youtubeKey="youtubeKey" title="title" resume="resume" release_date="18/10/2018"  />
        );
        expect(wrapper).toMatchSnapshot();
    });
    it("VideoDetail Sans video", () => {
        const wrapper = mount(
            <VideoDetail title="title" resume="resume" release_date="18/10/2018"  />
        );
        expect(wrapper).toMatchSnapshot();
    });
})