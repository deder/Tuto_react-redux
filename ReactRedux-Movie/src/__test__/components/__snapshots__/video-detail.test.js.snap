// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoDetail /> VideoDetail Avec video 1`] = `
<div
  className="row"
>
  <div
    className="col s12 m12 l12"
  >
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <i
          className="material-icons circle "
          style={
            Object {
              "backgroundColor": "#004D40",
              "color": "white",
              "marginRight": "10",
              "padding": "10",
            }
          }
        >
          video_library
        </i>
        <h5
          style={
            Object {
              "display": "inline",
            }
          }
        >
          title
        </h5>
      </div>
      <div
        className="card-image"
        style={
          Object {
            "padding": "10",
          }
        }
      >
        <span
          className=""
        >
          <div
            className={null}
            id={null}
          />
        </span>
      </div>
      <div
        className="card-content"
      >
        <span
          className="card-title"
        >
          Synopsis :
        </span>
        <p>
          resume
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`<VideoDetail /> VideoDetail Sans video 1`] = `
<div
  className="row"
>
  <div
    className="col s12 m12 l12"
  >
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <i
          className="material-icons circle "
          style={
            Object {
              "backgroundColor": "#004D40",
              "color": "white",
              "marginRight": "10",
              "padding": "10",
            }
          }
        >
          video_library
        </i>
        <h5
          style={
            Object {
              "display": "inline",
            }
          }
        >
          title
        </h5>
      </div>
      <div
        className="card-image"
        style={
          Object {
            "padding": "10",
          }
        }
      >
        <img
          alt="video vignette"
          src="http://www.azamedical.com/boutique/images_produits/no_image2-z.jpg"
          style={
            Object {
              "maxHeight": "390",
            }
          }
        />
      </div>
      <div
        className="card-content"
      >
        <span
          className="card-title"
        >
          Synopsis :
        </span>
        <p>
          resume
        </p>
      </div>
    </div>
  </div>
</div>
`;
